src/ResourceManager/Sql/Commands.Sql/packages.config

```diff
@@ -1,7 +1,10 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.Azure.KeyVault.Core" version="1.0.0" targetFramework="net45" />
-  <package id="Microsoft.Azure.Management.Sql" version="1.7.0-preview" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.Sql" version="1.8.0-preview" targetFramework="net452" />
   <package id="Microsoft.Azure.Management.Storage" version="2.4.0-preview" targetFramework="net45" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.10" targetFramework="net452" />
```

Comment:
@emgu-ms please remove the three references added in this file. These packages are already loaded elsewhere.